python/torch_mlir_e2e_test/reporting.py

```diff
@@ -310,7 +311,7 @@ def report_results(results: List[TestResult],
         results_by_outcome['XPASS']) != 0
 
     if had_unexpected_results:
-        print('\nUnexpected outcome summary:')
+        print(f'\nUnexpected outcome summary: ({config})')
```

Comment:
Please add/update test somewhere in here: https://github.com/llvm/torch-mlir/tree/main/python/test/torchscript_e2e_test

Reply:
LGTM! Thanks